src/AlarmClock.h

```diff
@@ -1,36 +1,50 @@
 /* 
  * File:   AlarmClock.h
- * Author: Craig Cogdill
- * Created: October 15, 2015 10:45am
+ * Author: Amanda Carbonari
+ * Created: January 5, 2016 4:35pm
+ * Description: Uses an "interruptable" std::thread, that checks every
+ *    microsecond to see if any of the two interrupt atomics have been set to
+ *    true (mReset, mExit). If they are it will stop it's sleep and then check
+ *    to see if it was a reset or exit. If it was neither (expired) it will busy
```

Comment:
"busy wait" implies no sleep. I'd rewrite something like: "If the alarm expires, then the AlarmClockInterruptableThread waits for reset or exit.